packages/mask/src/utils/fetcher.ts

```diff
@@ -35,6 +36,8 @@ function canAccessAsContent(url: string) {
         ['https://abs.twimg.com', 'https://upload.twitter.com'].includes(target.origin)
     )
         return true
+
+    if (location.origin.endsWith(EnhanceableSite.Mirror)) return true
```

Comment:
this does not make sense. do you mean: `url.endsWith(...) && location.origin.endsWith(...)`?